stellarphot/tests/test_core.py

```diff
@@ -146,31 +146,28 @@ def test_camera_unitscheck():
                              format='ecsv',
                              fast_reader=False)
 
+
 def test_base_enhanced_table_blank():
-    # This should raise error because you are attempting a blank data set
-    with pytest.raises(TypeError):
-        test_base = BaseEnhancedTable(test_descript)
+    # This should just return a blank BaseEnhancedTable
+    test_base = BaseEnhancedTable()
+    assert type(test_base) == BaseEnhancedTable
```

Comment:
The preferred idiom for type check is to use isinstance instead:

```suggestion
    assert isinstance(test_base, BaseEnhancedTable)
```